[2023-10-20 23:21:53,129] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-02-04T00:00:00+00:00 [queued]>
[2023-10-20 23:21:53,146] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-02-04T00:00:00+00:00 [queued]>
[2023-10-20 23:21:53,147] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:53,149] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:21:53,149] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:21:53,172] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): tercera_tarea> on 2023-02-04 00:00:00+00:00
[2023-10-20 23:21:53,178] {standard_task_runner.py:52} INFO - Started process 1940 to run task
[2023-10-20 23:21:53,183] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'tercera_tarea', 'scheduled__2023-02-04T00:00:00+00:00', '--job-id', '563', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpcjpo6ffx', '--error-file', '/tmp/tmp1dw89hbk']
[2023-10-20 23:21:53,185] {standard_task_runner.py:80} INFO - Job 563: Subtask tercera_tarea
[2023-10-20 23:21:53,290] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-02-04T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:21:53,394] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=tercera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-04T00:00:00+00:00
[2023-10-20 23:21:53,396] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:21:53,397] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2']
[2023-10-20 23:21:53,408] {subprocess.py:85} INFO - Output:
[2023-10-20 23:21:53,409] {subprocess.py:92} INFO - hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2
[2023-10-20 23:21:53,410] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:21:53,459] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=tercera_tarea, execution_date=20230204T000000, start_date=20231020T232153, end_date=20231020T232153
[2023-10-20 23:21:53,556] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:21:53,624] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
